modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -24,17 +24,22 @@ Use the following procedure to configure CD-ROMs for virtual machines.
 .Procedure
 
 . From the *{object-gui}* tab, select your {object}.
+
 . Select the *Overview* tab.
-. Click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
-* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
-* If there are CD-ROMs available, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+
+. To add or edit a CD-ROM configuration, click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
+
+* If CD-ROMs are unavailable for editing, the message displays: *The virtual machine doesn't have any CD-ROMs attached*.
+
+* If there are CD-ROMs available, you can remove a CD-ROM by clicking *-*.
+
 . In the *Edit CD-ROM* window, do the following:
-.. Select the type of CD-ROM configuration in the *Media Type* field.
-* On Windows systems, *Windows guest tools* is attached by default in
-the *Media Type* field.
-.. Complete the required information for each *Media Type*.
-.. Click *Add CD-ROM*.
-. When all CD-ROMs are added, click *Save*.
+
+* Select the type of CD-ROM configuration from the drop-down list for *Media Type*. CD-ROM configuration types are  *Container*, *URL* and *Persistent Volume Claim*.
+
+.. Complete the required information for each *Type*.
+
+.. When all CD-ROMs are added, click *Save*.
```

Comment:
Should this be steps a, b, and c? I think the structure of this step is confusing.

Reply:
Cleaned up the steps.